.azure/windows-stack.yml

```diff
@@ -1,41 +0,0 @@
-jobs:
```

Comment:
Agree in drop it temporary, but i would keep the config file and only remove the build from azure-pipelines.yaml

Reply:
You can revert this commit to bring it back